package.json

```diff
@@ -22,11 +18,15 @@
     "react-howler": "^3.7.4",
     "resolve-url-loader": "^3.0.1",
     "ts-loader": "^5.3.3",
-    "turbolinks": "^5.2.0",
-    "typescript": "^3.3.3"
+    "turbolinks": "^5.2.0"
   },
   "version": "0.1.0",
   "devDependencies": {
-    "webpack-dev-server": "^3.1.14"
+    "webpack-dev-server": "^3.1.14",
+    "@babel/preset-react": "^7.0.0",
+    "@types/react": "^16.8.3",
```

Comment:
Please remove react related stuff, we wanted to use react for player but then choose a different path.